File: tags/Release/SR-RBD/src/ddb/msg/client/TimeoutMessage.java
/**
 * 
 */
package ddb.msg.client;

import java.io.IOException;

import ddb.communication.DataInputStream;
import ddb.communication.DataOutputStream;
import ddb.msg.MessageType;

/** 
 * <!-- begin-UML-doc -->
 * <!-- end-UML-doc -->
 * @author User
 * @generated "UML to Java (com.ibm.xtools.transform.uml2.java5.internal.UML2JavaTransform)"
 */
public class TimeoutMessage extends ClientResponse {

	@Override
	public void fromBinary(DataInputStream s) throws IOException {
		// empty
		
	}

	@Override
	public MessageType getType() {
		return MessageType.CLIENT_TIMEOUT;
	}

	@Override
	public void toBinary(DataOutputStream s) throws IOException {
		// empty
	}
}

File: C#/RBD/RBD/Msg/Client/TimeoutMessage.cs
// +

using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

using RBD.Communication;

namespace RBD.Msg.Client
{
    class TimeoutMessage : ClientResponse
    {
        override
        public void FromBinary(DataInputStream s) //throws IOException 
        {
            // empty
        }

        override
        public MessageType GetMessageType()
        {
            return MessageType.CLIENT_TIMEOUT;
        }

        override
        public void ToBinary(DataOutputStream s) //throws IOException 
        {
            // empty
        }
    }
}
